### DISCUSSION TITLE
Como obter a conquista de tubarão no GitHub?

### DISCUSSION THREAD

**Discussion by LucasHavranekValor** [community/community#55945] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Gostaria de saber como obter a conquista de tubarão no GitHub?

**Comment by LucasHavranek** [comment, marked as answer]
Para obter a conquista de tubarão você deve criar um pull request em um repositório público ou privado e o pull request deve ser aceito para merge pelo proprietário do repositório.

**Comment by Gabrielbprado** [comment]
Tem como Perder a Medalha ?

**Reply by LucasHavranek** [reply]
Até o momento eu não perdi a conquista, não sei dizer se existe um prazo de validade.
